src/Database/Migrations/2022-10-20-182737_ShieldOAuth.php

```diff
@@ -54,6 +55,15 @@ public function up(): void
         ];
 
         $this->forge->addColumn('users', $fields);
+
+        $fields2 = [
+            'username'     => [
+                'type'     => 'VARCHAR(100)',
+                'null'     => true
+            ]
+        ];
+
+        $this->forge->modifyColumn('users', $fields2);
```

Comment:
If a developer has already run the migration, executing it again will have no effect because the migration system only detects new migrations.  

Therefore, **it's best to create a new migration file** and apply the necessary changes there, according to what you have done. This ensures that your changes are properly applied to the database without modifying previous migrations.

Reply:
I don't think that is a problem though. Any developer already using the library will already have handled this short field somehow; if anything, adjusting their table could break their fix. 

I think it is something only necessary for a new installation and so updating the existing migration is all that is needed.